vision/classification_and_detection/python/models/resnext50/anchor_utils.py

```diff
@@ -0,0 +1,131 @@
+import math
+import torch
+from torch import nn, Tensor
+
+from typing import List, Optional
+from models.resnext50.image_list import ImageList
```

Comment:
I'd avoid including the model source in the benchmark app and store torchscript in the model file - makes thing much easier to maintain.